torchgen/model.py

```diff
@@ -187,13 +187,14 @@ class DispatchKey(Enum):
     # END autogenerated
 
     def __str__(self) -> str:
-        return self.name
+        return self.name.replace("PrivateUse1", PrivateUse1Name.custom_name)
```

Comment:
No. I don't want this patched. Revert this change, and then modify the call sites you want translated one by one.